### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by soyfestivo**

### CODE REVIEW

File: src/canvastools/ts/ct.ts [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-import { PointRegion } from "./CanvasTools/Region/Point/PointRegion";
-import { RectRegion } from "./CanvasTools/Region/Rect/RectRegion";
-import { AreaSelector } from "./CanvasTools/Selection/AreaSelector";
-import { SelectionMode } from "./CanvasTools/Interface/ISelectorSettings";
-import { InvertFilter, GrayscaleFilter, BlurDiffFilter, ContrastFilter,
-         BrightnessFilter, SaturationFilter } from "./CanvasTools/CanvasTools.Filter";
-import { Rect } from "./CanvasTools/Core/Rect";
+import { Editor as CTEditor } from "./CanvasTools/CanvasTools.Editor";
+import { BlurDiffFilter, BrightnessFilter, ContrastFilter, GrayscaleFilter,
+         InvertFilter, SaturationFilter } from "./CanvasTools/CanvasTools.Filter";
```
Comment (pritamDas9191) [inline src/canvastools/ts/ct.ts:3]: can we add the linter rule that adds these on each line separately when the max length exceeds ?
Would look cleaner.

File: src/canvastools/ts/CanvasTools/Region/RegionsManager.ts [hunk 2]
```diff
@@ -173,7 +178,9 @@ export class RegionsManager {
         }
         this.sortRegionsByArea();
         this.redrawAllRegions();
-        document.getElementById("regionAnnouncer").innerHTML = tagsDescriptor.toString();
+        if (this.regionAnnouncer) {
```
Comment (pritamDas9191) [inline src/canvastools/ts/CanvasTools/Region/RegionsManager.ts:181]: was this causing a  NullReferenceException earlier ?

Comment (soyfestivo) [reply src/canvastools/ts/CanvasTools/Region/RegionsManager.ts:181]: Yep

File: .github/workflows/CI.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          node-version: ${{ matrix.node-version }}
+      - name: Install yarn
+        run: npm install yarn -g
+      - name: Install packages
+        run: yarn install
+      - name: Check for code linter errors
+        run: yarn lint
+      - name: Run tests
+        run: yarn test
+      - name: Run tests
```
Comment (pritamDas9191) [inline .github/workflows/CI.yml:32]: why twice?

Comment (soyfestivo) [reply .github/workflows/CI.yml:32]: My mistake, I'll remove it 👍

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+    "editor.codeActionsOnSave": {
+        "source.fixAll.eslint": true,
+        "source.fixAll.tslint": true,
+        "source.fixAll.stylelint": true
+    },
+    "git.enableCommitSigning": true
```
Comment (pritamDas9191) [inline .vscode/settings.json:7]: add this too git.alwaysSignOff ?

File: src/canvastools/ts/ct.ts (hunk 1 above)
Comment (soyfestivo) [reply src/canvastools/ts/ct.ts:3]: Looks like this isn't actually a tslint rule but a pretier rule. I'm going to just stick with tslint for now.

File: .vscode/settings.json (hunk 4 above)
Comment (soyfestivo) [reply .vscode/settings.json]: You got it

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,9 @@
 ## Changelog
 
+### 2.2.17
+* Add `.updateRegionVisibility()` to `RegionManager` for easier region visibility toggling
+* Fix a bug with add region accessability announcing
```
Comment (pritamDas9191) [inline CHANGELOG.md:5]: nit: accessibility

**Review by pritamDas9191 — DISMISSED** [review]

**Review by pritamDas9191 — APPROVED** [review]

**Pull request merged**